NAMESPACE

```diff
@@ -23,12 +23,12 @@ export(radsub)
 export(radtr)
 export(radtte)
 export(radvs)
+export(rcd_var_relabel)
```

Comment:
do we have to export it at all?

Reply:
let's remove it from exports then
There are a few more funs like this to be removed. Conceptually, this pacakge should export only data creation funs and nothing more than that.